BankApplication.cpp

```diff
@@ -0,0 +1,59 @@
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <iomanip>
+#include "Banking.cpp"
```

Comment:
Do not include .cpp files, declare classes and functions in .h files and include them